typings/index.d.ts

```diff
@@ -3081,7 +3081,9 @@ declare module 'discord.js' {
     guildUpdate: [oldGuild: Guild, newGuild: Guild];
     inviteCreate: [invite: Invite];
     inviteDelete: [invite: Invite];
+    /** @deprecated Use messageCreate instead */
```

Comment:
Are these visible anywhere except when looking at these lines in this file?

Reply:
They're only visible when you look up this property in the types, but doesn't show up when using `client.on('message', cb);` and others.